dags/symbol_set_cleanup.py

```diff
@@ -0,0 +1,152 @@
+import datetime
+import dagster
+from django.db.models import Q
+from django.utils import timezone
+
+from dags.common import JobOwners
+from posthog.models.error_tracking.error_tracking import ErrorTrackingSymbolSet
+
+
+class SymbolSetSelectionConfig(dagster.Config):
+    days_old: int = 30
+    delete_unused: bool = False  # Delete symbol sets with null last_used
+
+
+class SymbolSetDeletionConfig(dagster.Config):
+    dry_run: bool = False
+
+
+@dagster.asset
+def symbol_sets_to_delete(config: SymbolSetSelectionConfig) -> list[ErrorTrackingSymbolSet]:
+    cutoff_date = timezone.now() - datetime.timedelta(days=config.days_old)
```

Comment:
I guess you don't need to keep track of timezones/utc here as it is a rolling process, so checking just to make sure.